pkg/composer/up.go

```diff
@@ -86,7 +87,26 @@ func (c *Composer) Up(ctx context.Context, uo UpOptions, services []string) erro
 		return err
 	}
 
-	return c.upServices(ctx, parsedServices, uo)
+	if err := c.upServices(ctx, parsedServices, uo); err != nil {
+		return err
+	}
+
+	orphans, err := c.getOrphanContainers(ctx, parsedServices)
```

Comment:
This operation might be slow and should be called only when `RemoveOrphans` is true

Reply:
Hi @AkihiroSuda, I appreciate this operation could introduce additional delays into the `compose up` workflow, but without performing it we cannot warn users that orphaned containers exist:

```go
	if numOrphans > 0 {
		if uo.RemoveOrphans {
			if err := c.downContainers(ctx, orphans, true); err != nil {
				return fmt.Errorf("error removing orphaned containers: %s", err)
			}
		} else {
			logrus.Warnf("%d orphaned containers exist. Run with --remove-orphans to remove them.", numOrphans)
		}
	}
```

This would be a deviation from the behaviour of `docker-compose`, which prints a similar warning. 

We can probably reduce the time taken without losing this error message by comparing the total number of containers in the `parsedServices` slice with the total number of running containers in the compose project; this would remove the requirement to call the `.Labels()` method for each container:

```go
	for _, container := range containers {
		containerLabels, err := container.Labels(ctx)
		if err != nil {
			return false, err
		}
		if name == containerLabels[labels.Name] {
			// container exists
			return true, nil
		}
	}
```

Please let me know if you're happy with that approach, and I'll make the changes accordingly.

Reply:
> This would be a deviation from the behaviour of docker-compose, which prints a similar warning.

I think this difference is acceptable.

Reply:
ping @fhke

Reply:
Hey @AkihiroSuda, apologies for the delay. I've updated this PR with the requested changes.